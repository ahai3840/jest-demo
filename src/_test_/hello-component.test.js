import React from 'react';
import toJson from 'enzyme-to-json';
import HelloComponent from '../hello-component';

describe('testing hello component',()=>{
  let wrapper;
   beforeEach(()=>{
      wrapper = shallow(<HelloComponent name="enzyme" />);
   })
   test('render hello component',()=>{
      expect(toJson(wrapper)).toMatchSnapshot();
   })
   test('hello component has className hello ',()=>{
    expect(wrapper.find('.hello').length).toBe(1);
   })
});