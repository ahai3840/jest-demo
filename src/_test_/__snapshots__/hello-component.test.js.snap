// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing hello component render hello component 1`] = `ShallowWrapper {}`;
